modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -283,6 +180,90 @@ where `pod` is the name of a machine config daemon pod.
 
 ... Resolve any errors in the logs shown by the output from the previous command.
 
+. To start the migration, configure the OVN-Kubernetes cluster network provider by using one of the following commands:
+
+** To specify the network provider without changing the cluster network IP address block, enter the following command:
++
+[source,terminal]
+----
+$ oc patch Network.config.openshift.io cluster \
+  --type='merge' --patch '{ "spec": { "networkType": "OVNKubernetes" } }'
+----
+
+** To specify a different cluster network IP address block, enter the following command:
++
+[source,terminal]
+----
+$ oc patch Network.config.openshift.io cluster \
+  --type='merge' --patch '{
+    "spec": {
+      "clusterNetwork": [
+        {
+          "cidr": "<cidr>",
+          "hostPrefix": "<prefix>"
+        }
+      ]
+      "networkType": "OVNKubernetes"
+    } 
+  }'
+----
++
+where `cidr` is a CIDR block and `prefix` is the slice of the CIDR block apportioned to each node in your cluster. You cannot use any CIDR block that overlaps with the `100.64.0.0/16` CIDR block, because the OVN-Kubernetes network provider uses this block internally.
++
+[IMPORTANT]
+====
+You cannot change the service network address block during the migration.
+====
+
+. Wait until the Multus daemon set rollout completes.
```

Comment:
It's a bit confusing that this step says to "wait" but is followed by a command. Can you clarify this by saying something like, "Verify that the Multus daemon set rollout is complete:"?